src/components/StatusStrip/StatusStrip.tsx

```diff
@@ -0,0 +1,68 @@
+import { useState, useEffect, ReactNode } from "react";
+
+import "./_status-strip.scss";
+
+type Status = {
+  label: string;
+  count: number;
+};
+
+type Props = {
+  statusList: {
+    model: Status[];
+  };
+};
+
+export default function StatusStrip({ statusList }: Props) {
+  type Status = {
+    groupName: string;
+    statement: ReactNode;
+    count: number;
+  };
+  const [status, setStatus] = useState<Status>({
+    groupName: "",
+    statement: [],
+    count: 0,
+  });
+
+  useEffect(() => {
+    let proposedStatus: Status = {
+      groupName: "",
+      statement: [],
+      count: 0,
+    };
+    let count = 0;
+
+    function incrementCount(incrementBy: number) {
+      count = count + incrementBy;
+    }
+
+    Object.entries(statusList).forEach(([groupName, statuses]) => {
+      const statement =
+        statuses &&
+        Object.values(statuses).map((status) => {
+          incrementCount(status.count);
+          return (
+            <div className="p-chip" key={status.label}>
```

Comment:
Should this use the `Chip` component from `react-components` now? It would need some work to support this content but it would help with some of the interaction logic.